boards/arm/mimxrt1015_evk/doc/index.rst

```diff
@@ -0,0 +1,170 @@
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/arm/mimxrt1015_evk/mimxrt1015_evk_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and I/Os
+====================
+
+The MIMXRT1015 SoC has five pairs of pinmux/gpio controllers.
+
++---------------+-----------------+---------------------------+
+| Name          | Function        | Usage                     |
++===============+=================+===========================+
+| GPIO_AD_B0_05 | GPIO            | LED                       |
++---------------+-----------------+---------------------------+
+| GPIO_EMC_09   | GPIO            | SW4                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_06 | LPUART1_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_07 | LPUART1_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The MIMXRT1015 SoC is configured to use the 24 MHz external oscillator on the
+board with the on-chip PLL to generate a 500 MHz core clock.
+
+Serial Port
+===========
+
+The MIMXRT1015 SoC has eight UARTs. ``LPUART1`` is configured for the console,
+and the remaining are not used.
+
+Programming and Debugging
+*************************
+
+The MIMXRT1015-EVK includes the :ref:`nxp_opensda` serial and debug adapter
+built into the board to provide debugging, flash programming, and serial
+communication over USB.
+
+To use the Segger J-Link tools with OpenSDA, follow the instructions in the
+:ref:`nxp_opensda_jlink` page using the `Segger J-Link OpenSDA V2.1 Firmware`_.
+The Segger J-Link tools are the default for this board, therefore it is not
+necessary to set ``OPENSDA_FW=jlink`` explicitly when you invoke ``make
+debug``.
+
+With these mechanisms, applications for the ``mimxrt1015_evk`` board
+configuration can be built and debugged in the usual way (see
+:ref:`build_an_application` and :ref:`application_run` for more details).
+
+The pyOCD tools do not yet support this SoC.
+
+Flashing
+========
+
+The Segger J-Link firmware does not support command line flashing, therefore
+the usual ``flash`` build system target is not supported.
+Instead, see the https://www.nxp.com/docs/en/application-note/AN12108.pdf for flashing instructions.
```

Comment:
Instead of a bare URL, how about: 
```suggestion
Instead, see the `How to Enable Boot from QSPI Flash Application note
<https://www.nxp.com/docs/en/application-note/AN12108.pdf>`_ for flashing instructions.
```

Reply:
This section has been updated according to the PR #14000.